modules/era/lua/era_status_effects.lua

```diff
@@ -0,0 +1,18 @@
+-----------------------------------
+-- Common Requires
+-----------------------------------
+require('modules/module_utils')
+-----------------------------------
+-- Module definition
+-----------------------------------
+-- This module attempts to have a centralized file to override status effects with era changes.
+local eraStatusEffects = Module:new('era_status_effects')
```

Comment:
So, the problem here is that there's no way to selectively choose overrides within a module. If all the era status changes go in here, it'll mix up all the different eras. I guess for now it would make sense to change this to `era_composure.lua` and `era_composure` module name, and leave it in this folder. At some point later when we have more era modules someone else can start to organise them into folders based on period etc.

Reply:
I went with `era_effect_composure.lua`

Wemay end up being the ones organizing the folder structure, so in case theres more effect modules down the line, I figure having them all together will make things slighty easier